python/scripts/meta.json

```diff
@@ -2083,6 +2083,558 @@
+                "sql_name": "applicant_race_3",
+                "type": "text"
+            },
+            {
+                "display_name": "applicant_race_4",
+                "display_text": "",
+                "source_name": "applicant_race_4",
+                "sql_name": "applicant_race_4",
+                "type": "text"
+            },
+            {
+                "display_name": "applicant_race_5",
+                "display_text": "",
+                "source_name": "applicant_race_5",
+                "sql_name": "applicant_race_5",
+                "type": "text"
+            },
+            {
+                "display_name": "applicant_race_name_1",
+                "display_text": "",
+                "source_name": "applicant_race_name_1",
+                "sql_name": "applicant_race_name_1",
+                "type": "text"
+            },
+            {
+                "display_name": "applicant_race_name_2",
+                "display_text": "",
+                "source_name": "applicant_race_name_2",
+                "sql_name": "applicant_race_name_2",
+                "type": "text"
+            },
+            {
+                "display_name": "applicant_race_name_3",
+                "display_text": "",
+                "source_name": "applicant_race_name_3",
+                "sql_name": "applicant_race_name_3",
+                "type": "text"
+            },
+            {
+                "display_name": "applicant_race_name_4",
+                "display_text": "",
+                "source_name": "applicant_race_name_4",
+                "sql_name": "applicant_race_name_4",
+                "type": "text"
+            },
+            {
+                "display_name": "applicant_race_name_5",
+                "display_text": "",
+                "source_name": "applicant_race_name_5",
+                "sql_name": "applicant_race_name_5",
+                "type": "text"
+            },
+            {
+                "display_name": "applicant_sex",
+                "display_text": "",
+                "source_name": "applicant_sex",
+                "sql_name": "applicant_sex",
+                "type": "text"
+            },
+            {
+                "display_name": "applicant_sex_name",
+                "display_text": "",
+                "source_name": "applicant_sex_name",
+                "sql_name": "applicant_sex_name",
+                "type": "text"
+            },
+            {
+                "display_name": "application_date_indicator",
+                "display_text": "",
+                "source_name": "application_date_indicator",
+                "sql_name": "application_date_indicator",
+                "type": "text"
+            },
+            {
+                "display_name": "as_of_year",
+                "display_text": "",
+                "source_name": "as_of_year",
+                "sql_name": "as_of_year",
+                "type": "text"
```

Comment:
The values in this field will consistently be a 4-digit year.  I wasn't sure if they should be dates or not, so I cast them as text.